cnnimg.py

```diff
@@ -31,10 +31,7 @@
 import Image as img
 import numpy as np
 import os.path
-import warnings
 
```

Comment:
Well, I turned the prints into () syntax as that's backwards compatible with python2 and doesn't break on python3, but I couldn't get PIL (Image) to work on Python 3, so I guess that wasn't necessary.